catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -430,8 +433,8 @@ public Response getDocument(@PathParam("sourceid") String sourceid, @PathParam("
         URI absolutePath = uriInfo.getAbsolutePath();
         MultivaluedMap<String, String> map = uriInfo.getQueryParameters();
 
-        if (id != null) {
-            LOGGER.debug("Got id: {}", id);
+        if (encodedId != null) {
+            LOGGER.debug("Got id: {}", encodedId);
```

Comment:
Nit- all these separate debug statements should be merged together and cleaned up.